files/openmw.appdata.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2015 Alexandre Moine <nobrakal@gmail.com> -->
+<application>
+ <id type="desktop">openmw.desktop</id>
+ <metadata_license>CC0-1.0</metadata_license>
+ <project_license> GPL-3.0 and MIT and zlib</project_license>
+ <name>OpenMW</name>
+ <summary>Unofficial open source engine re-implementation of the game Morrowind</summary>
+ <description>
+  <p>
+  OpenMW is a recreation of the engine for the popular role-playing game Morrowind by Bethesda Softworks. You need to own and install the original game for OpenMW to work.
+  </p>
+  <p>
+  The main quests in Morrowind, Tribunal and Bloodmoon are all completable. Some issues with side quests are to be expected (but rare).
+  </p>
+  <p>
+  Pre-existing modifications created for the original Morrowind engine can be hit-and-miss, but a mod created for Morrowind may not necessarily run in OpenMW.
```

Comment:
Not too happy about the duplication of readme.txt. If we really need a long description, please write one that isn't going to get outdated.